data/hai/hai reveal 4 middle.txt

```diff
@@ -1412,33 +1412,236 @@ mission "Hai Reveal [A11] Sound the Alarm"
 		has "Hai Reveal [A10] Demanding Satisfaction: done"
 	on offer
 		event "hai and human ships at wormhole"
-		log `Teeneep has been rejected by the Elders again, but has set in a motion a plan which has triggered an election instead. This snap election has closed the border and risked creating a diplomatic incident.`
+		log `You left Teeneep with an uncertain fate while you were ushered out of the Elder Council chambers by a panicked Sayari. Sayari warned of an impending snap election to which panicked Elders will close the border and risk a diplomatic incident.`
```

Comment:
I'm gonna take a look at A11 myself later, but to raise this up first, no, the logs are _not_ written in the second person. This is written by the player character themselves.

Reply:
Thank you for clarifying that! The log in the first person now. How does it look?

Reply:
And it should be one of the only usage of past tense in the game.

Reply:
Yes, I wrote it in past tense (I just checked).